sgx-hasher/src/lib.rs

```diff
@@ -79,6 +80,52 @@ impl Hasher {
     }
 }
 
+pub fn sign(
+    author: sig::Author,
+    contents: sig::Contents,
+    key: rsa::Rsa<pkey::Private>,
+) -> std::io::Result<sig::Signature> {
```

Comment:
I'm wondering if it is better to use the `openssl` `ErrorStack` error type here...

Reply:
@lkatalin This still needs to be fixed.